conftest.py

```diff
@@ -19,6 +20,16 @@ def disable_celery_until_we_are_ready(settings):
     settings.QPC_ENABLE_CELERY_SCAN_MANAGER = False
 
 
+@pytest.fixture(autouse=True)
+def default_data_dir(settings, tmp_path: Path) -> Path:
+    """Set a clean data dir for storing files."""
+    data_dir = tmp_path / "data"
+    settings.DEFAULT_DATA_DIR = data_dir
+    settings.LOG_DIRECTORY = data_dir / "logs"
```

Comment:
Instead of monkey-patching settings directly, did you consider the `override_settings` context manager from `django.test`? That way it cleans up after itself if we ever _don't_ want this fixture (I did noticed the `autouse=true` above).

Reply:
huh. I thought the settings fixture was exactly the same thing as override_settings

Reply:
> This fixture **[settings]** will provide a handle on the Django settings module, and automatically revert any changes made to the settings (modifications, additions and deletions).

https://pytest-django.readthedocs.io/en/latest/helpers.html#settings

Reply:
Ah, I overlooked that you were using the settings fixture, not the settings module. Got it!

Reply:
We actually do want the `autouse` behavior because tests will mess with the default path.
They already were. They were reading `./quipucords/quipucords/secret.txt`, for instance.

I believe all places that can store data should be hermetically isolated in tests to avoid mixing local development files.